src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/PassForm.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@(Html
+    .DevExtreme()
+    .Form<LoginModelRecord>()
+    .OnInitialized("passFormInitialized")
+    .ID("PassForm")
+    .Items(items =>
+    {
+        items.AddCustomFormTextBox(model => model.Password);
+
+        items

```

Comment:
Crete Reusable component for Previous and Next

Reply:
Done